ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -177,7 +215,7 @@ class ServicesHomeScreen extends React.Component<Props, State> {
           <ScreenContentHeader
             title={I18n.t("services.title")}
             icon={require("../../../img/icons/services-icon.png")}
-            fixed={true}
+            fixed={Platform.OS === "ios"}
```

Comment:
It comes back :grin: https://github.com/teamdigitale/io-app/pull/1269#discussion_r318028493

Reply:
shouldn't it be as follow?
```
<ScreenContentHeader
      title={I18n.t("services.title")}
      icon={require("../../../img/icons/services-icon.png")}
      fixed={Platform.OS === "ios"}
 />
```

Reply:
yes you're right, sorry. Forget it :+1: